### ISSUE TITLE
fixed_dx

### ISSUE THREAD

**Issue by peltoglacier** [OGGM/oggm#786]
I like using a fixed_dx for inversion for comparability between glaciers, however I am having trouble getting the model to respect this. 

When I use:
`gdirs = workflow.init_glacier_regions(rgidf, reset=True, force=True, from_prepro_level=3) `

It seems from_prepro_level nullifies my 'fixed_dx', even though when I print:
`cfg.PARAMS['grid_dx_method']`
it returns
'fixed'

However my each gdirs have a different dx.

If I remove from_prepro_level, I get a message: " please download and process some climate data" When calling: utils.get_ref_mb_glaciers(gdirs)

**Comment by peltoglacier** [comment]
Ah, I removed from_prepro_level added in:

`# Preprocessing and climate tasks
task_list = [
    tasks.glacier_masks,
    tasks.compute_centerlines,
    tasks.initialize_flowlines,
    tasks.compute_downstream_line,
    tasks.compute_downstream_bedshape,
    tasks.catchment_area,
    tasks.catchment_intersections,
    tasks.catchment_width_geom,
    tasks.catchment_width_correction,
    tasks.process_cru_data,
#     tasks.local_t_star,
#     tasks.mu_star_calibration,
]
for task in task_list:
    execute_entity_task(task, gdirs)`

This appears to have solved my issue

**Comment by fmaussion** [comment]
> Ah, I removed from_prepro_level

Yes, this is it. To understand what's going on: `from_prepro_level` will download [already prepared, pre-processed glacier directories](https://docs.oggm.org/en/stable/input-data.html#pre-processed-directories). This is the recommended way to run OGGM as it ensures that we all work with the same data, but it has downsides: certain parameters like grid spacing (or which DEM to use) are decided very early in the model chain and are therefore fixed in the pre-processed directories. 

 What you are doing now is that you are relying on the raw data sources, i.e. the original DEMS and are doing your own cuisine.
